src/relay/blockchain/events_informations.py

```diff
@@ -0,0 +1,133 @@
+import math
+from collections import namedtuple
+from typing import List
+
+from relay.blockchain.currency_network_events import (
+    BalanceUpdateEvent,
+    BalanceUpdateEventType,
+    TrustlineUpdateEventType,
+)
+from relay.network_graph.interests import calculate_interests
+
+InterestAccrued = namedtuple("InterestAccrued", ["value", "interest_rate", "timestamp"])
```

Comment:
If there is no good reason to use named tuples, please consider using the attrs module. The attrs documentation lists some reasons why you should use that: https://www.attrs.org/en/stable/why.html#namedtuples

Reply:
I don't think the reasons listed are good enough to make me use attrs. 

> "If you want a tuple with names, by all means: go for a namedtuple." 

Yes.

But I'm biased by the fact that when we use attrs my editor complains that every use of it is unexpected.

Reply:
pycharm should support it: https://blog.jetbrains.com/pycharm/2018/06/pycharm-2018-2-eap-2-2/ 

If that is your only reason to keep using the namedtuple, please change it.